maven-surefire-plugin/src/site/resources/xsd/surefire-test-report.xsd

```diff
@@ -18,6 +18,11 @@
   ~ under the License.
   -->
 <xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema">
+    <xs:simpleType name="SUREFIRE_TIME">
+        <xs:restriction base="xs:string">
+            <xs:pattern value="(([0-9]{0,3},)*[0-9]{3}|[0-9]{0,3})*(\.[0-9]{0,3})?"/>
```

Comment:
I think this pattern is also not correct because the fractions separator should be (.) and not (,). The formatter uses English, see class `ReporterUtils`. WDYT?
```
NumberFormat numberFormat = NumberFormat.getInstance( Locale.ENGLISH );
return numberFormat.format( runTime / MS_PER_SEC );
```